.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
Did the prettier rules on the `eslint` config not work for you?

Reply:
Yeah, had to make some changes to my editor in between. They're working now. :)